src/main/kotlin/bitbucket/BitbucketClient.kt

```diff
@@ -72,6 +88,25 @@ class BitbucketClient(
         }
     }
 
+    fun retrieveMergeStatus(pr: PR): MergeStatus {
+        return try {
+            val urlBuilder = mergeUrl(pr)
+            val request = httpRequestFactory.createGet(urlBuilder.toUrlString())
+            val mergeStatus = sendRequest(request, mergeStatusResponseHandler)
+            mergeStatus.unknown = false
+            mergeStatus
+        } catch (e: Exception) {
+            listener.requestFailed(e)
+            MergeStatus(false, false, listOf(Veto("Request Error", "")))
```

Comment:
Minor. Could we replace list with *emptyList()*

Reply:
My idea was that we display "Request Error" as a tooltip for merge button in this case, so user could have some idea why it is disabled